android/src/main/java/nl/fhict/happynews/android/manager/PostManager.java

```diff
@@ -118,7 +122,7 @@ public void refresh(String query, final Context context, final FutureCallback<Pa
      */
     private void loadPage(int page, int size, Context context, final FutureCallback<Page> callback) {
         Ion.with(context)
-            .load(apiUrl + "/post?page=" + page + "&size=" + size)
+            .load(apiUrl + "/post?page=" + page + "&size=" + size + "&whitelist=" + generateWhitelist(context))
```

Comment:
The old code is not quite the way it should be. We might want to make this the same as for example line 140.